tutorials/discussions/plot_background_filtering.py

```diff
@@ -21,22 +21,21 @@
 Problem statement
 =================
 
-The practical issues with filtering electrophysiological data are covered
-well by Widmann *et al.* in [7]_, in a follow-up to an article where they
-conclude with this statement:
+Practical issues with filtering electrophysiological data are covered
+in Widmann *et al.* (2015) [7]_, where they conclude with this statement:
 
     Filtering can result in considerable distortions of the time course
-    (and amplitude) of a signal as demonstrated by VanRullen (2011) [[3]_].
+    (and amplitude) of a signal as demonstrated by VanRullen (2011) [3]_.
```

Comment:
I don't really get it.

Reply:
We are inserting the text. Inserted text should be enclosed in `[]`. If at some point NumpyDoc changes how the `[3]_` gets rendered (e.g., adds an option to render as `(3)` or superscript 3, then we have then mis-quoted the authors because our editorial addition is no longer enclosed in `[]`. Plus the entire `[3]`-with-link is what we are trying to add, so `[[3]_]` makes sense from this perspective, too.

Reply:
Alright. I wouldn't handle this and the other quotes so strictly. I do agree that you can't just change an original quote, but in most (if not all) places we don't have to use full quotes, I'd prefer if we formulate in our own words and then add the reference. For example, a missing whitespace is a typo and too minor to add [sic], so I'd rather insert the whitespace to correct it and get rid of the literal quote. Same for the [[3]] link, it just looks ugly. But anyway, I'm not insisting on these changes.

Reply:
Also, unless my PDF search functionality is broken, this paragraph does not appear in the Widmann et al. (2015) article.

Reply:
Ahh yes it's in 2012 not 2015